### PR TITLE
Champions Projects Wrap-Up Blog Post

### PR DESCRIPTION
**Pull request by yabellini**
Skeleton
Authors?

### CODE REVIEW

**Comment by yabellini** [comment]
@steffilazerte this is ready for you review. :-)

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+-   **Marcos Prunello** from Argentina, worked on his package [karel](https://mpru.github.io/karel/), which implements Karel the robot in R, a useful tool to teach introductory concepts about programming in an interactive and fun way, by writing programs for Karel to complete tasks. Marcos worked on improving error messages and develop a infrastructure that allow the package to provides functions, arguments, messages, and documentation in both English and Spanish.  The multilingual solution enables additional languages to be added. The package [is in the process of review](https://github.com/ropensci/software-review/issues/620), being the first to have a **bilingual review**!. His mentor was [Lukas Wallrich](author/lukas-wallrich/) from the UK. 
+
+
+### Developing new packages
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:35]:
```suggestion
### Projects developing new packages
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+The other type of project was to develop a new package.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:37]:
```suggestion
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+-   **Haydee Svab**, from Brazil, developed a new package called [odbr](https://cran.r-project.org/web/packages/odbr/index.html) that allows accessing and processing data from the Brazilian origin and destination (od) surveys and implements many of the algorithms that Haydee developed for her master's thesis. She has the goal to publish the package on CRAN, and after several attemps, and with the support of her mentor, the package was [published](https://cran.r-project.org/web/packages/odbr/index.html. 
+
+-   **Bilikisu Wunmi Olatunji** from Nigeria, developed the new package [ChartkickR](https://github.com/BWOlatunji/chartkickR). The package is an htmlwidget that seamlessly integrates with the Chartkick JavaScript library. It provides R users with a straightforward interface to generate beautiful and interactive charts using R code. The package provides ten types of plots that you can accomplish with very few lines of code. [You can read more in Bilikisu's blog post](/blog/).
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:41]: Will there be a blog post, or should we remove that for now?
```suggestion
-   **Bilikisu Wunmi Olatunji** from Nigeria, developed the new package [chartkickR](https://github.com/BWOlatunji/chartkickR). The package is an htmlwidget that seamlessly integrates with the Chartkick JavaScript library. It provides R users with a straightforward interface to generate beautiful and interactive charts using R code. The package provides ten types of plots that you can produce with very few lines of code. [You can read more in Bilikisu's blog post](/blog/).
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+-   **Ghozayel Elotteebi** from Saudi Arabia, worked on a new package called [Lextale](https://ghozayel.github.io/Lextale/). To Ghozayel knowledge, this is the first R package in the field of linguistics developed in Saudi Arabia and would be the first local contribution to rOpenSci. She is working on fixing functionalities on the package to send to the review process. 
+
+
+## Outreach Activities
+
+Our champions carried out a wide range of outreach activities like talks, conference papers, workshops, blog articles, community calls and meetups. The objective of these activities was to share what they did and learned during the program with their communities beyond rOpenSci.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:48]:
```suggestion
Our Champions carried out a wide range of outreach activities like talks, conference papers, workshops, blog articles, community calls and meetups. The objective of these activities was to share what they did and learned during the program with their communities beyond rOpenSci.  
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+## Outreach Activities
+
+Our champions carried out a wide range of outreach activities like talks, conference papers, workshops, blog articles, community calls and meetups. The objective of these activities was to share what they did and learned during the program with their communities beyond rOpenSci.  
+
+Outreach activities were carried out from May 2023 to December 2023 involving local communities (universities, RUGs, R-Ladies Chapters, conferences, and organizations) and delivered in Champions’ native language (English, Spanish, and Portuguese). 
+
+### Workshops
+
+Paola Corrales developed and delivered the online workshop for non-selected applicants on [Git and GitHub](https://paocorrales.github.io/git-developing-software-together/) already giving back to the program. She will teach this workshop for the second cohort.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:54]:
```suggestion
Paola Corrales developed and delivered an online workshop for non-selected applicants on [Git and GitHub](https://paocorrales.github.io/git-developing-software-together/) already giving back to the program. She will also teach this workshop for the second cohort.
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+{{< figure src = "pao-git-github.png" alt = "Paola Corrales sharing a slide about the git workshop" caption = "<strong>Paola Corrales teaching Git for the Champions Program</strong>">}}
+
+Marcos Prunello developed a workshop on [how to create R packages and make them multilingual](https://mpru.github.io/charla_renrosario_2023/crear_paq_multiling.html) in Spanish for an in-person meetup hosted with the National University of Rosario (Argentina) and RenRosario R User Group, reactivating the activities of this local community after a hiatus for the pandemic.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:58]:
```suggestion
Marcos Prunello developed a workshop on [how to create R packages and make them multilingual](https://mpru.github.io/charla_renrosario_2023/crear_paq_multiling.html) in Spanish for an in-person meetup hosted with the National University of Rosario (Argentina) and RenRosario R User Group, reactivating the activities of this local community after a hiatus during the pandemic. 
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+{{< figure src = "pao-git-github.png" alt = "Paola Corrales sharing a slide about the git workshop" caption = "<strong>Paola Corrales teaching Git for the Champions Program</strong>">}}
+
+Marcos Prunello developed a workshop on [how to create R packages and make them multilingual](https://mpru.github.io/charla_renrosario_2023/crear_paq_multiling.html) in Spanish for an in-person meetup hosted with the National University of Rosario (Argentina) and RenRosario R User Group, reactivating the activities of this local community after a hiatus for the pandemic. 
+
+{{< figure src = "RenRosario.jpeg" alt = "Marcos Prunello and the attendees to his workshop on multilingual packages" caption = "<strong>Attendees to Marcos Prunello's workshop on Multilingual Packages</strong>">}}
+
+Bilikisu Wunmi Olatunji delivered a workshop on _Working with JavaScript in R_ for the R-Ladies Abuja chapter. She have to learn this topic for the development of her package for the Champions Program.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:62]:
```suggestion
Bilikisu Wunmi Olatunji delivered a workshop on _Working with JavaScript in R_ for the R-Ladies Abuja chapter. She learned about this topic for the development of her package for the Champions Program.
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+{{< figure src = "r-ladies-abuja-meetup.jpeg" alt = "Workshop flyer for the Abuja R-Ladies Chapter featuring Bilikisu" caption = "<strong>R-Ladies Abuja flyer for the Workshop Working with JavaScript in R</strong>">}}
+
+### Community Calls and Co-working sessions
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:66]:
```suggestion
### Community Calls and Coworking sessions
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+
+Carolina Pradier, Victor Ordu, Paola Corrales, Haydee Svab (champions), Beatriz Milz, Elio Campitelli, and Adam Spark (mentors) participated in [Community Calls](/commcalls/july2023-championprogram/) and [Co-working sessions ](https://ropensci.org/coworking/) for the champions program. They shared their experience, their work, and tips for applying to the program. They also spoke with and joined the community manager in talks with local groups about the program to encourage other people to participate. 
+
+{{< figure src = "R-Ladies-BASR-Meetup.png" alt = "A post in social media that shows Pao Corrales sharing her experience being a champions at the R-Ladies Buenos Aires and Santa Rosa meetup. The zoom meeting shows a slides with the training topics of the champions program" caption = "<strong>Pao Corrales sharing her experience being a champions at the R-Ladies Buenos Aires and Santa Rosa meetup</strong>">}}
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:70]:
```suggestion
{{< figure src = "R-Ladies-BASR-Meetup.png" alt = "A post in social media that shows Pao Corrales sharing her experience being a champions at the R-Ladies Buenos Aires and Santa Rosa meetup. The zoom meeting shows a slides with the training topics of the champions program" caption = "<strong>Pao Corrales sharing her experience being a Champion at the R-Ladies Buenos Aires and Santa Rosa meetup</strong>">}}
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+
+Carolina Pradier also organized [an event online in Spanish](https://vimeo.com/899372049) to get together the community of users and developers of her package eph. In this event Carolina updated the community about all the improvements she made to the package under the champions program and three panelist explain how they use the package.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:72]:
```suggestion
Carolina Pradier also organized [an online event in Spanish](https://vimeo.com/899372049) for the community of users and developers of her package eph. In this event Carolina updated the community about all the improvements she made to the package through the Champions Program and three panelists explained how they use the package.
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+
+
+### Talks
+
+Our champions are part of other communities like R-Ladies, LatinR and R User Groups. They organize severals activities with these groups.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:77]:
```suggestion
Our Champions are part of other communities like R-Ladies, LatinR and local R User Groups, and organized several activities with these groups. 
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+
+{{< figure src = "cesar-talk.png" alt = "Event flyer. Featuring Cesar headshor picture and his talk title Preparing remote sensing data for deep learning using r-spatial. May 8. 9:00-10:00 UTC+1 - Register on Eventbrite." caption = "<strong>Cesar's talk event flyer</strong>">}}
+
+Marcos Prunello talks about [rOpenSci](https://mpru.github.io/charla_renrosario_2023/presentacion_ropensci.html) in Spanish was part of a meetup with RenRosario RUG and the Universidad Nacional de Rosario and Carolina Pradier delivered a talk about her project [Paquete {eph}: Novedades e ideas para su uso](https://caropradier.github.io/slides_eph_ropensci/#1). 
+
+{{< figure src = "flyers.png" alt = "Events flyers in Spanish. eph event featuring Valentin, Vladimiro and Guido talks titles in Spanish.  Second flyer shows a short bio about Marcos and information on the event venue in the Universidad Nacional de Rosario. Marcos' talk title Que es rOpenSci y como ser parte." caption = "<strong>eph and RenRosario event flyers in Spanish</strong>">}}
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:85]:
```suggestion
{{< figure src = "flyers.png" alt = "Events flyers in Spanish. First flyer shows a short bio about Marcos and information on the event venue in the Universidad Nacional de Rosario. Marcos' talk title Que es rOpenSci y como ser parte. Second flyer for the eph event featuring talks by Valentin, Vladimiro and Guido. " caption = "<strong>RenRosario and eph event flyers in Spanish</strong>">}}
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+
+Haydee Svab and Beatriz Milz organized an in person event with R-Ladies Sao Paulo in portuguese [Pacotes em R de dados brasileiros](https://r-ladies-sao-paulo.github.io/2023-11-pacotes-br/) where Haydee delivered a talk about the rOpenSci Champions Program and the odbr package she developed as her project in the program. The event also highlight others women package developers from Brazil. This blog post (in [Portuguese](https://rladies-sp.org/posts/2023-12-evento-pacotes/) and [English](https://rladies-sp.org/posts/2023-12-evento-pacotes/english.html)) summarize the event and share the slides of all the talks.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:87]:
```suggestion
Haydee Svab and Beatriz Milz organized an in person event with R-Ladies Sao Paulo in portuguese [Pacotes em R de dados brasileiros](https://r-ladies-sao-paulo.github.io/2023-11-pacotes-br/) where Haydee delivered a talk about the rOpenSci Champions Program and the odbr package she developed as her project in the program. The event also highlighted others women developers from Brazil. This blog post (in [Portuguese](https://rladies-sp.org/posts/2023-12-evento-pacotes/) and [English](https://rladies-sp.org/posts/2023-12-evento-pacotes/english.html)) summarizes the event and shares the slides of all the talks. 
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+
+Haydee Svab and Beatriz Milz organized an in person event with R-Ladies Sao Paulo in portuguese [Pacotes em R de dados brasileiros](https://r-ladies-sao-paulo.github.io/2023-11-pacotes-br/) where Haydee delivered a talk about the rOpenSci Champions Program and the odbr package she developed as her project in the program. The event also highlight others women package developers from Brazil. This blog post (in [Portuguese](https://rladies-sp.org/posts/2023-12-evento-pacotes/) and [English](https://rladies-sp.org/posts/2023-12-evento-pacotes/english.html)) summarize the event and share the slides of all the talks. 
+
+{{< figure src = "R-Ladies-SaoPaulo.jpg" alt = "R-Ladies Sao Paulo attendees' group photo. The gropu is around 30 people smiling. Haydee is wearing an R-Ladies T-shirt" caption = "<strong>R-Ladies Sao Paulo event group photo</strong>">}}
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:89]:
```suggestion
{{< figure src = "R-Ladies-SaoPaulo.jpg" alt = "R-Ladies Sao Paulo attendees' group photo. The group is around 30 people smiling. Haydee is wearing an R-Ladies T-shirt" caption = "<strong>R-Ladies Sao Paulo event group photo</strong>">}}
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+
+All the latin american participant of the program [wrote a summary about their expericience on the program](https://ropensci.org/es/blog/2024/01/09/champions-program-latinr/) that was accepted at LatinR and [presented in Universidad de La Republica (Uruguay) by Haydee Svab in Portuguese representing all the authors](https://hsvab.github.io/latinr2023-championsprogram/#/). 
+
+All our champions and mentors wrote blog posts for the rOpenSci webpage ([English](https://ropensci.org/tags/champions-program/), [Spanish](https://ropensci.org/es/tags/champions-program/)). 
+
+## Closing the program and next steps
+
+Our champions have been very successful in the delivery of their projects, building high quality software with focus on their countries and discipline. They work impact also spread to their home institutions and related communities that they participate in. Not only because of the outreach activities sharing information and knowledge contextualized for their audience, but also for researching and developing solutions to common problems for participation such as language.
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:97]:
```suggestion
It's bittersweet to say goodbye to our 2022-2023 Champions. Our Champions have been very successful in the delivery of their projects, building high quality software with a focus on their countries and discipline. The impact of their work also spread to their home institutions and related local communities. Not only through their outreach activities sharing information and knowledge contextualized for their audience, but also through their researching and developing solutions to common barriers to participation such as language. 
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 16]
```diff
@@ -0,0 +1,105 @@
...
+
+{{< figure src = "R-Ladies-SaoPaulo.jpg" alt = "R-Ladies Sao Paulo attendees' group photo. The gropu is around 30 people smiling. Haydee is wearing an R-Ladies T-shirt" caption = "<strong>R-Ladies Sao Paulo event group photo</strong>">}}
+
+All the latin american participant of the program [wrote a summary about their expericience on the program](https://ropensci.org/es/blog/2024/01/09/champions-program-latinr/) that was accepted at LatinR and [presented in Universidad de La Republica (Uruguay) by Haydee Svab in Portuguese representing all the authors](https://hsvab.github.io/latinr2023-championsprogram/#/). 
+
+All our champions and mentors wrote blog posts for the rOpenSci webpage ([English](https://ropensci.org/tags/champions-program/), [Spanish](https://ropensci.org/es/tags/champions-program/)). 
+
+## Closing the program and next steps
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:95]: Closing sounds like it won't run again 🙀 
```suggestion
## Saying goodbye to our Champions
```

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 17]
```diff
@@ -0,0 +1,105 @@
...
+
+Our champions have been very successful in the delivery of their projects, building high quality software with focus on their countries and discipline. They work impact also spread to their home institutions and related communities that they participate in. Not only because of the outreach activities sharing information and knowledge contextualized for their audience, but also for researching and developing solutions to common problems for participation such as language. 
+
+Most of our champions will continue engage with rOpenSci, they already have activities planned for 2024. Stay tuned to learn about them!
```
Comment (steffilazerte) [inline content/blog/2024-03-22-champions-program-projects/index.md:99]:
```suggestion
However, many of our Champions will continue engage with rOpenSci, and even have already planned activities for 2024. Stay tuned to learn about them! 
```

**Review by steffilazerte — COMMENTED** [review]
Hi @yabellini, I really like this post! It's so cool to hear about all the awesome things the Champions have done. 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2024-03-22-champions-program-projects/index.md [hunk 18]
```diff
@@ -0,0 +1,105 @@
...
+
+-   **Alican Cagri Gokcek** from Turkey [reviewed Victor’s package](https://github.com/ropensci/software-review/issues/600). As a blind developer, Cagri add a focus on the accessibility of the visualization the package created in his review. His mentor was [Elio Campitelli](/author/elio-campitelli/) from Argentina. 
+
+-   **Cesar Aybar** from Peru, developed the [rgeeExtra](https://r-earthengine.com/rgeeExtra/index.html) package that extends the functionality of {rgee}, a package to access Google Earth Engine API, to be able to access and process remote sensing spatial data with a friendly syntax for R users. Cesar worked on adding functionality, change the sintaxis of the package, improve test coverage and error messages, and replaced spatial dependencies that will be removed. His package is [under review](https://github.com/ropensci/software-review/issues/608). His mentor was [Marc Choisy](/author/marc-choisy/) from Vietnam.
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2024-03-22-champions-program-projects/index.md:28]: Yes, it is rgeeExtra. And ok to change the link.  Thanks!!

File: content/blog/2024-03-22-champions-program-projects/index.md (hunk 3 above)
Comment (yabellini) [reply content/blog/2024-03-22-champions-program-projects/index.md:41]: There is already published; I will add the link.

File: content/blog/2024-03-22-champions-program-projects/index.md (hunk 16 above)
Comment (yabellini) [reply content/blog/2024-03-22-champions-program-projects/index.md:95]: Ohhh, got it!

**Pull request merged**